Decorator/TextDecorator/MessageDecorator.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Decorator
+{
+    abstract class MessageDecorator : Message
```

Comment:
Why do we need this file? We can simple inheirit from the `Message` directly, right?